server/src/test/java/org/elasticsearch/index/fielddata/IndexFieldDataServiceTests.java

```diff
@@ -212,6 +215,27 @@ public void onRemoval(ShardId shardId, String fieldName, boolean wasEvicted, lon
         ifdService.clear();
     }
 
```

Comment:
Maybe add a test like `testFieldDataCacheListener` that checks that we retrieve a single entry in the cache when accessing multiple fields in the json field ?

Reply:
👍